src/ffi/c.rs

```diff
@@ -468,7 +468,7 @@ mod c_backend {
     #[cfg(feature = "zlib-ng")]
     const ZLIB_VERSION: &str = "2.1.0.devel\0";
     #[cfg(all(not(feature = "zlib-ng"), feature = "zlib-rs"))]
-    const ZLIB_VERSION: &str = "1.3.0-zlib-rs-0.5.0\0";
+    const ZLIB_VERSION: &str = "1.3.0-zlib-rs-0.5.1\0";
```

Comment:
I just realised that if it wasn't for this string, `flate2` wouldn't need a release if `zlib-rs` increases its patch level.
Something I am wondering about is if this lock-step releasing is truly required for everything to work?

Reply:
As far as I can tell, the only thing that `zlib-rs` actually cares about is that [the first byte of the string is a '1' character](https://github.com/trifectatechfoundation/zlib-rs/blob/bd501f88a03aacf99163ba93431d870ae66865b5/libz-rs-sys/src/lib.rs#L1740).